.circleci/config.yml

```diff
@@ -92,55 +92,45 @@ jobs:
 
   frontend_test:
     docker:
-      - image: recipeyak/base:v3
+      - image: circleci/node:12.4.0
```

Comment:
maybe this will load faster because CircleCI will have these cached on their nodes. 🤞